README.md

```diff
@@ -68,6 +73,30 @@ The rate limit is calculated as follows:
 
 This is a linear rate limit that increases over time with a maximum limit. This rate limit is derived from these values which can be set by and admin OR updated by the `CONTROLLER` role. The `CONTROLLER` updates these values to increase/decrease the rate limit based on the functionality within the contract (e.g., decrease the rate limit after minting USDS by the minted amount by decrementing `lastAmount` and setting `lastUpdated` to `block.timestamp`).
 
+## Trust Assumptions
+Below are all stated trust assumptions for using this contract in production:
+- The `DEFAULT_ADMIN_ROLE` is fully trusted, to be run by governance.
+- The `RELAYER` role is assumed to be able to be fully compromised by a malicious actor. 
+  - The logic in the smart contracts must prevent the movement of value anywhere outside of the ALM system of contracts.
+  - The rate limits setup must prevent the possibility of griefing or locking/burning value by a malicious actor.
+  - The `FREEZER` must be able to stop the compromised `RELAYER` from performing more harmful actions by using the `freeze()` function.
+- AAVE deposits are subject to AAVE governance changes, such as setting reserves inactive, paused or frozen. 
+- USDC holdings are subject to blacklist risk.
+- Ethena deposits are subject to USDe depeg risks, blacklist risks, and mint and burn limits.
+- The Ethena Minter is fully trusted. If it is compromised it can: 
+  - Accept orders submitted by a malicious delegated signer that would burn a large amount of USDC for a small amount of USDe (bad quote).
+  - DOS minting/burning of USDe, locking funds.
+- A compromised `RELAYER` can DOS Ethena unstaking, but this can be mitigated by freezing the Controller and reassigning the `RELAYER`. This is outlined in a test `test_compromisedRelayer_lockingFundsInEthenaSilo`.
+
+## Operational Requirements
+- All ERC-4626 vaults that are onboarded MUST have an initial burned shares amount that prevents rounding-based frontrunning attacks. These shares have to be unrecoverable so that they cannot be removed at a later date.
+- All ERC-20 tokens are to be non-rebasing with sufficiently high decimal precision.
+- Rate limits must be configured for specific ERC-4626 vaults and AAVE aTokens (vaults without rate limits set will revert). Unlimited rate limits can be used as an onboarding tool.
+- Rate limits must take into account:
+  - Risk tolerance for a given protocol
+  - Griefing attacks (e.g., repetitive transactions with high slippage by malicious relayer).
+  - Reputational attack vectors (e.g., sudden withdraws from utilization-based markets spiking interest rates).
```

Comment:
This is less of a concern. The withdrawal issue is more if there is a delay. For example, being able to withdraw 1b sUSDe all at once will not cause slippage or even reputational damage, but it will carry a huge opportunity cost. Spiking interest rates is less of a concern for the purposes of this.